src/components/menu/templates/main.tsx

```diff
@@ -41,7 +41,8 @@ const items = {
     'Advanced': '/docs/theming/advanced',
   },
   'Navigation': {
-    'Angular': '/docs/navigation/angular'
+    'Angular': '/docs/navigation/angular',
```

Comment:
Note for when this is ready - changing urls here to /docs/react/navigation etc...